protocol-tests/src/test/java/fixtures/dpgupdate/DpgUpdateTests.java

```diff
@@ -80,7 +80,7 @@ public void postParameters() {
         requestOptions.setHeader("content-type", "image/jpeg");
 //        requestOptions.setHeader("content-length", String.valueOf(parameter.getLength()));
 
-        response = client2.postParametersWithResponse(parameter, requestOptions);
+        response = client2.postParametersWithResponse(parameter.getLength(), parameter, requestOptions);
```

Comment:
Why does not method not having a `contentType` parameter?

Reply:
I also notice this and I checked for m4 output, in m4 output, it does not have content-type parameter. m4 output for this operation is as below:
```
- operationId: params_postParameters
        apiVersions:
          - version: 1.0.0
        parameters:
          - *ref_2
        requestMediaTypes:
          application/json: &ref_16
            parameters:
              - &ref_13
                schema: *ref_12
                implementation: Method
                required: true
                language:
                  default:
                    name: parameter
                    description: 'I am a body parameter with a new content type. My only valid JSON entry is { url: "http://example.org/myimage.jpeg" }'
                protocol:
                  http:
                    in: body
                    style: json
              - schema: *ref_4
                implementation: Method
                origin: modelerfour:synthesized/accept
                required: true
                language:
                  default:
                    name: accept
                    description: Accept header
                    serializedName: Accept
                protocol:
                  http:
                    in: header
            signatureParameters:
              - *ref_13
            language:
              default:
                name: ''
                description: ''
            protocol:
              http:
                path: /serviceDriven/parameters
                method: post
                knownMediaType: json
                mediaTypes:
                  - application/json
                uri: '{$host}'
          image/jpeg: &ref_17
            parameters:
              - &ref_15
                schema: *ref_14
                implementation: Method
                required: true
                language:
                  default:
                    name: parameter
                    description: 'I am a body parameter with a new content type. My only valid JSON entry is { url: "http://example.org/myimage.jpeg" }'
                protocol:
                  http:
                    in: body
                    style: binary
              - schema: *ref_4
                implementation: Method
                origin: modelerfour:synthesized/accept
                required: true
                language:
                  default:
                    name: accept
                    description: Accept header
                    serializedName: Accept
                protocol:
                  http:
                    in: header
            signatureParameters:
              - *ref_15
            language:
              default:
                name: ''
                description: ''
            protocol:
              http:
                path: /serviceDriven/parameters
                method: post
                binary: true
                knownMediaType: binary
                mediaTypes:
                  - image/jpeg
                uri: '{$host}'
        requests:
          - *ref_16
          - *ref_17
        signatureParameters: []
        responses:
          - schema: *ref_6
            language:
              default:
                name: ''
                description: Answer from service
            protocol:
              http:
                knownMediaType: json
                mediaTypes:
                  - application/json
                statusCodes:
                  - '200'
        language:
          default:
            name: PostParameters
            description: POST a JSON or a JPEG
        protocol: {}
```